lib/nimbase.h

```diff
@@ -208,11 +214,7 @@ __AVR__
 #    define N_FASTCALL_PTR(rettype, name) rettype (*name)
 #    define N_SAFECALL_PTR(rettype, name) rettype (*name)
 #  endif
-#  ifdef __cplusplus
-#    define N_LIB_EXPORT  extern "C"
-#  else
-#    define N_LIB_EXPORT  extern
-#  endif
+#  define N_LIB_EXPORT NIM_EXTERNC __attribute__((visibility("default")))
```

Comment:
This needs to be in a ``if !defined(_MSC_VER)`` or similar.

Reply:
it's already in the else branch of `#if defined(WIN32) || defined(_WIN32) /* only Windows has this mess... */`  as you can see by clicking expand for more context in github UI or looking the diff locally

Reply:
Oh alright, I misread. (I actually did click and expanded the view.)